wasmaudioworklet/audioworkletpolyfill.js

```diff
@@ -6,12 +6,21 @@
  */
 
 if(typeof AudioContext !== 'function') {
-    window.AudioContext = window.webkitAudioContext;
+    /** @type {any} */
+    const win = window
+    window.AudioContext = win.webkitAudioContext;
```

Comment:
I used `any` here so TS won't complain about `window.webkitAudioContext` being a non-existent property. We don't care much about the polyfill types, the end user doesn't care about it.